examples/docs_snippets/docs_snippets/concepts/assets/graph_backed_asset.py

```diff
@@ -76,7 +76,7 @@ def middle_asset(upstream_asset):
     return add_one(upstream_asset)
 
 
-middle_asset = AssetsDefinition.from_graph(middle_asset)
+middle_asset = AssetsDefinition.from_graph(middle_asset)  # type: ignore
```

Comment:
why is this a type error?

Reply:
Because the type of the variable `middle_asset` starts out as `GraphDefinition` from the first declaration but then gets changed to `AssetsDefinition`. Redefining the type of a local variable is a type error if the new type is not a subset of the old type.